packages/odyssey-design-tokens/src/font.json

```diff
@@ -11,21 +11,19 @@
       }
     },
     "size": {
-      "base": { "value": "16" },
-      "scale": { "value": 1.1487 },
-      "body": {
-        "lede": { "value": "" },
-        "base": { "value": "" },
-        "caption": { "value": "" }
+      "base": { "value": "16px" },
+      "scale": {
```

Comment:
Alternately, something like as rec'd by @arnoldsandoval-okta:

```
"Neg1": { "value": "0.871rem" },
"0": { "value": "1rem" },
"1": { "value": "1.149rem" },
"2": { "value": "1.32rem" },
"3": { "value": "1.516rem" },
"4": { "value": "1.741em" },
"5": { "value": "2rem" }
```

In common use, most folks should be using `font.size.caption` rather than our negative "palette" value here.

Reply:
Don't love Neg1 here or really the numbers either because the scale isn't linear even if the labels are. Consider using the same labels that we're already using for space as they have less connotation about the delta between entries and we can keep adding without worrying about negative numbers: 

```
        "xxs": { "value": "0.871rem" },
        "xs": { "value": "1rem" },
        "s": { "value": "1.149rem" },
        "m": { "value": "1.32rem" },
        "l": { "value": "1.516rem" },
        "xl": { "value": "1.741em" },
        "xxl": { "value": "2rem" }
```

Reply:
The values aren't linear in our type scale, but the scale itself is ordinal. That is, a designer will want to know how many "steps" from baseline each size is.

Ex: "medium" here would be the intended size of an H3, and our base font-size would be considered "extra small".

Currently, the smallest possible size of the scale will always be one less than our base font size, to prevent any serious a11y mishaps. From there, the next step up is Base. Then we proceed with proportional values up the Heading chain until we reach our targeted maximum font size.

That said, we're no longer using a generated modular scale. I have a feeling Visual Design will not want to return to the old setup now that we have a manually curated implementation. Perhaps we could take an approach rather different from Color and Spacing, instead only serving up "functional" tokens for typography:

- font.size.base
- font.size.caption/sub
- font.size.lede
- font.size.title.6, 5, 4, 3, 2, 1

@josesolano-okta your thoughts here would be v helpful. I'm happy to talk this through live as well.